README.md

```diff
@@ -60,6 +60,7 @@ Service Name | Artifact Coordinates
 [IAM Policy Management](https://cloud.ibm.com/apidocs/iam-policy-management?code=java) | com.ibm.cloud:iam-policy-management:0.48.2
 [IBM Cloud Shell](https://cloud.ibm.com/apidocs/cloudshell?code=java) | com.ibm.cloud:ibm-cloud-shell:0.48.2
 [Open Service Broker](https://cloud.ibm.com/apidocs/resource-controller/ibm-cloud-osb-api?code=java) | com.ibm.cloud:open-service-broker:0.48.2
+[Partner Usage Reports](https://cloud.ibm.com/apidocs/partner-apis/partner-usage-reports?code=java) | com.ibm.cloud:partner-usage-reports:0.48.2
```

Comment:
This link to the online reference docs is different than the one in the Go SDK.  These should probably be the same.